docker-compose.physical.controller-board.yml

```diff
@@ -17,9 +17,7 @@ services:
     devices:
       - /dev/ttyACM0:/dev/ttyACM0
     environment:
+      DRIVERCONFIG_VIRTUALCONTROLLERBOARD: "false"
```

Comment:
Don't we want this to be `true` since user will not have the controller board?

Reply:
The docker-compose physical will only be used if you have the Arduino board

Reply:
Awe, missed that. THX!